### PR TITLE
Solved 4th task

### PR DESCRIPTION
**Pull request by PatrykKuniczak**
I will solved it with express and ejs as a "frontend"

User can:
- Add recipes,
- Add products with amount,
- Remove recipes
- Remove products

Every operation is protected by relation in SQL, when user remove recipe every product, which been used inside, been remove.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        host: "localhost",
+        user: "root",
+        port: 3310,
+        password: "pass123",
+    })
+
+    await db.query(`CREATE DATABASE IF NOT EXISTS cook_assistant`);
+    await db.query(`USE cook_assistant`);
+
+    createRecipeTable(db);
```
Comment (16pxdesign) [inline src/app.js:22]: Application code should not create database.

File: src/database/tables.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const createRecipeTable = (db) => {
+    db.query(`CREATE TABLE IF NOT EXISTS recipes (
```
Comment (16pxdesign) [inline src/database/tables.js:2]: where is .sql file

**Review by 16pxdesign — APPROVED** [review]
Haven't tested on my pc but looks ok.

File: src/database/tables.js (hunk 2 above)
Comment (PatrykKuniczak) [reply src/database/tables.js:2]: Nowhere, i creating everything in code, like a ORM, i I treat it as a entity :)

File: src/app.js (hunk 1 above)
Comment (PatrykKuniczak) [reply src/app.js:22]: I use it to make easier, test it on your PC, but if u didn't test it, i can remove it before next PR.

File: src/database/tables.js (hunk 2 above)
Comment (16pxdesign) [reply src/database/tables.js:2]: That was not a task. You need it for later tasks. Orm generates sql migrations based on code for example. It's different approach

File: src/app.js (hunk 1 above)
Comment (16pxdesign) [reply src/app.js:22]: Export to  .sql file with next task

Comment (16pxdesign) [reply src/app.js:22]: Sql file will be executte one manually it's not needed to check it every time app starts (time wasting during boot). I'm not surethis or previous task mentioned create sql file

**Pull request merged**